STL_Extension/doc/STL_Extension/CGAL/algorithm.h

```diff
@@ -115,17 +115,17 @@ namespace CGAL {
 /*!
 \ingroup STLAlgos
 
-\deprecated This function is deprecated. `std::next` should be used
+\deprecated This function is deprecated. `std::next()` should be used
 instead.
 
 
-Returns the next iterator, i.e.
-the result of `operator++` on a forward iterator.
+Returns the next iterator, i.e.,
+the result of `operator++()` on a forward iterator.
 
 
 \sa `CGAL::predecessor()`
 
-\returns `++it`.
+\returns `it`.
```

Comment:
Not sure about that change; maybe just remove the `\returns` and simply write "calls `operator++()` on `it` and returns `it` afterward"?

In any case, `predecessor()` should also have these changes.

Reply:
Or maybe returns `++it`  ?

Reply:
but that's what it was before...?

Reply:
It returns the successor of `it` without incrementing `it`

Reply:
predecessor

Reply:
> It returns the successor of `it` without incrementing `it`

The point was that "returns `it`" is wrong.

Maybe the simplest would be to just say this functions is `std::next(it)`?